docs/DeveloperGuide.md

```diff
@@ -300,32 +300,50 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `TutorEase` and the **Actor** is the `user`, unless specified otherwise)
+
+**Use Case: UC01 - Add student contact**
+
+**MSS**:
+1. Tutor keys in required fields to add student contact
+2. TutorEase adds the student contact
```

Comment:
Perhaps end every sentence with . to follow the current DG

Reply:
Okay sure, I will change it!